docs/getting-started.md

```diff
@@ -157,6 +157,12 @@ as soon as it's mounted and will rerun it when the query or variables change.
 
 [Read more about the result's API in the Architecture's Results section.](architecture.md#operation-results)
 
+#### Watch `useQuery` React hook screencast on egghead
+
+[Query Graphql Data With Urql Using React Hooks][egghead]
```

Comment:
I'd opt for just having it inline and more prominently featured :) can you put it above the "Read more about..." link and make it a quote, which we use for notes? e.g.

```md
> A tutorial on the `useQuery` hook is also available as a [screencast on Egghead](https://egghead.io/lessons/graphql-query-graphql-data-with-urql-using-react-hooks).
```

(Inline links are completely fine since this is not reused :+1: )

Reply:
Sounds great!